frontend/src/__tests__/cypress/cypress/support/commands/odh.ts

```diff
@@ -339,14 +352,149 @@ declare global {
+        response: OdhResponse<GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/recurringruns/:recurringRunId`,
+        options: { path: { namespace: string; recurringRunId: string } },
+        response: OdhResponse<PipelineRunJobKFv2>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `DELETE /api/service/pipelines/:namespace/dspa/apis/v2beta1/recurringruns/:recurringRunId`,
+        options: { path: { namespace: string; recurringRunId: string } },
+        response: OdhResponse<GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/pipelines/create`,
+        options: { path: { namespace: string }; times?: number },
+        response: OdhResponse<PipelineKFv2 | GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/pipelines/upload`,
+        options: { path: { namespace: string }; times?: number },
+        response: OdhResponse<PipelineKFv2 | GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs`,
+        options: { path: { namespace: string } },
+        response: OdhResponse<ListPipelineRunsResourceKF | GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs`,
+        options: { path: { namespace: string }; times?: number },
+        response: OdhResponse<PipelineRunKFv2 | GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `DELETE /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs/:runId`,
+        options: { path: { namespace: string; runId: string } },
+        response: OdhResponse<GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs/:runId`,
+        options: { path: { namespace: string; runId: string } },
+        response: OdhResponse<PipelineRunKFv2 | GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/runs/:runId`,
+        options: { path: { namespace: string; runId: string } },
+        response: OdhResponse<GoogleRpcStatusKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `GET /api/service/pipelines/:namespace/dspa/apis/v2beta1/experiments`,
+        options: { path: { namespace: string } },
+        response: OdhResponse<ListExperimentsResponseKF>,
+      ): Cypress.Chainable<null>;
+
+      interceptOdh(
+        type: `POST /api/service/pipelines/:namespace/dspa/apis/v2beta1/experiments/:experimentId`,
```

Comment:
The `dspa` segment needs to be parameterized as it represents the name of the service.